plugins/gimp/file-jxl-save.cc

```diff
@@ -643,6 +675,52 @@ bool JpegXlSaveOpts::SetPrecision(int gimp_precision) {
   return true;
 }  // JpegXlSaveOpts::SetPrecision
 
+class GimpParser {
+ public:
+  JxlFrameHeader LayerNameToFrameHeader(gchar* layer_name) {
+    JxlFrameHeader frame_header;
+    JxlEncoderInitFrameHeader(&frame_header);
+    gchar** remainder = &layer_name;
+    LayerNameToJxlLayerName(layer_name, remainder);
+    if (g_str_has_prefix(*remainder, " (")) {
+      gchar* remainder2;
+      gchar** endptr = &remainder2;
+      gint64 ms_len = g_ascii_strtoll(*remainder + 2, endptr, 10);
+      if (ms_len == 0) {
+        ms_len = 1;
```

Comment:
Why force frames to have a duration of at least 1 ms? JXL does support zero-duration frames and specifies that all consecutive zero-duration frames should be blended before showing a frame. While GIF/WebP do not support this (they have a minimum frame duration for historical reasons, which is longer than 1ms btw), JXL does and it seems useful to allow using this feature.